deploy/olm-catalog/jaeger-operator/manifests/jaeger-operator.clusterserviceversion.yaml

```diff
@@ -23,6 +23,7 @@ metadata:
     support: Jaeger Community
   name: jaeger-operator.v1.25.0
   namespace: placeholder
+  operators.openshift.io/infrastructure-features: '["disconnected"]'
```

Comment:
What does this exactly mean? that the operator can work in a cluster without internet connectivity? 

I have found only this https://docs.openshift.com/container-platform/4.5/operators/operator_sdk/osdk-generating-csvs.html

Reply:
https://docs.openshift.com/container-platform/4.6/operators/operator_sdk/osdk-generating-csvs.html

Reply:
Yes -- this means that the operator can run on a private network without access to the public internet.

Reply:
This annotation seems to be misplaced: it's under metadata, should be under annotations.